### DISCUSSION TITLE
Os comandos .net não esta funcionando via terminal Windows

### DISCUSSION THREAD

**Discussion by ghost** [balta-io/forum#125] [Tira Dúvidas] [answered]
Pessoal, quando eu digito o seguinte comando conforme a aula do professor no terminal 

`dotnet new console`

logo em seguida da o seguinte erro ou aviso 

**The command could not be loaded, possibly because:
  * You intended to execute a .NET application:
      The application 'new' does not exist.
  * You intended to execute a .NET SDK command:
      No .NET SDKs were found.

Download a .NET SDK:
https://aka.ms/dotnet/download

Learn about SDK resolution:
https://aka.ms/dotnet/sdk-not-found**

Alguém por favor poderia me dizer como resolver? Obrigada

**Comment by EdsonSantosActuar** [comment, marked as answer]
Fala @DeniseBarbosa!

Tente executar o comanda `dotnet --version` em seu teminal, o resultado esperado, é receber a versão atual que está instalado em sua máquina.
Em geral o erro que você recebeu é a falta do SDK, 
Acredito que será necessário a reinstalação, ao término da instalação você pode usar o comando mencionado ou o `dotnet --info` que trará todos os SKDs e rutimes instalados em sua máquina. 

Espero ter ajudado e bons estudos!!
